.github/workflows/release.yml

```diff
@@ -23,6 +23,9 @@ jobs:
       - name: Clear existing docker image cache
         run: docker image prune -af
 
+      - name: Setup Gradle Build Action
+        uses: gradle/gradle-build-action@v2
+
       - name: Run Gradle Build
         run: ./gradlew build --scan --no-daemon -i -x test
 
```

Comment:
We can also switch to using the `gradle-build-action` for execution, but it probably changes nothing. 

https://github.com/gradle/gradle-build-action#gradle-execution

Reply:
that's useful when testing with different gradle versions. In our case would not help that much given we use gradle wrapper